base/show.jl

```diff
@@ -1686,7 +1686,14 @@ end
 
 ## AST printing ##
 
-show_unquoted(io::IO, val::SSAValue, ::Int, ::Int)      = print(io, "%", val.id)
+function show_unquoted(io::IO, val::SSAValue, ::Int, ::Int)
+    if get(io, :maxssaid, typemax(Int))::Int < val.id
+        color = :red # invalid SSAValue, print this in red for better recognition
+    else
+        color = :plain
```

Comment:
This will undo any formatting that is externally applied, I think this is better of as just a regular `print`.